cms/templates/emails/utilities_bulk_update_done_message.txt

```diff
@@ -0,0 +1,14 @@
+<%! from django.utils.translation import ugettext as _ %>
+${_("Hello, {user}").format(user=user_username)}
+
+% if success:
+${_("Your bulk update on the {course} course has successfully completed. Previously published problems have been automatically republished.").format(course=course_name)}
+% else:
+${_("Your bulk update on the {course} course has failed due to internal error. Problems in your course may be partially updated. Please contact our team for help.").format(course=course_name)}
```

Comment:
I suspect `due to internal error` doesn't add much insight to a non-technical user.
Perhaps we can strike it and just say "Your bulk update on the X/Y/Z course has failed."

Reply:
done